tests/bench_sizeof_coretypes/README.md

```diff
@@ -0,0 +1,6 @@
+# Measure Size of Core Types
+
+This application simply list the size of all types defined by the RIOT core.
+Its purpose is to provide a base for tracking the memory requirements of those
+types over time and to provide a simple way to judge the impacts of future core
+changes on memory usage.
```

Comment:
See comment in https://github.com/RIOT-OS/RIOT/pull/9348 made under the readme, about documenting future work eg in an issue. Ie., implementing memory requirements tracking over time, including maybe raising an alert if the memory requirements go up and an "extended review required" flag/label applied or something

Reply:
feel free. But not in scope of this PR.